Tribler/Core/CacheDB/SqliteCacheDBHandler.py

```diff
@@ -857,10 +849,10 @@ def freeSpace(self, torrents2del):
             return False
 
         # delete torrents from db
-        sql_del_torrent = u"UPDATE Torrent SET torrent_file_name = NULL WHERE torrent_id = ?"
+        sql_del_torrent = u"UPDATE Torrent SET name = NULL WHERE torrent_id = ?"
         # sql_del_tracker = "delete from TorrentTracker where torrent_id=?"
         # sql_del_pref = "delete from Preference where torrent_id=?"
-        tids = [(torrent_id,) for torrent_file_name, torrent_id, relevance, weight in res_list]
+        tids = [(torrent_id,) for name, torrent_id, relevance, weight in res_list]
```

Comment:
The torrent file name used to point to the actual collected torrent file on the filesystem. Since we moved to the LevelDB store, it would contain the infohash(?) if the torrent was present on the store or NULL if it wasn't. 

AFAIK It's not deleting the entry because there's other stuff pointing at it.

That's what I know about it. But I guess something is/should be deleting the actual torrent files from the torrent store too.